src/ConfigResource.php

```diff
@@ -184,17 +157,26 @@ public function fetch($tree = false)
      * @param  array $config
      * @return array
      */
-    public function replaceKey($keys, $value, array $config)
+    public function replaceKey($keys, $value, array $config = null)
     {
+        if (is_null($config)) {
+            $config = $this->getConfig();
+        }
+
         if (! is_array($keys)) {
             $keys = explode('.', $keys);
         }
 
         $key = array_shift($keys);
 
+        $haveKeys = (bool) count($keys) > 0;
```

Comment:
hm... I think we have here conversion twice, I mean it works like:
```
((bool) count($keys)) > 0
```
so first int -> bool and than comparison bool with int.

See:
https://3v4l.org/m0fnk
https://3v4l.org/TfFvI

I think we can just use `if ($keys)` (as we have in other places).

Reply:
That's fine with me.  Anything is better than the code that was there before.